src/main/java/Vendor.java

```diff
@@ -28,11 +29,12 @@ public Lotto buy(long money) {
 
     private LottoNumbers get() {
         Collections.shuffle(ALL_AVAILABLE_LOTTO_NUMBER_LIST);
+
         return new LottoNumbers(
                 ALL_AVAILABLE_LOTTO_NUMBER_LIST.stream().limit(LottoNumbers.SIZE).collect(Collectors.toList()));
     }
 
-    public Aggregator aggregate(ContainCounts containCounts) {
-        return new Aggregator(containCounts);
+    public BigDecimal yield(Ranks ranks) {
```

Comment:
`Vendor`의 내부 변수를 사용하는게 없어보이는데, `수익율` 계산을 `Vendor`에 넣은 이유가 궁금합니다 :thinking:

Reply:
현실에서 로또를 구매하는 흐름을 생각해보니까 

- Vendor -> 로또를 직접 구매(편의점 등)
- Lottery - > 로또 추첨, 공시(로또 기금 재단)

로 대략 역할을 나눌 수 있겠구나 생각했어요. 
이러한 이유로 Vendor 는 '나' 에게 로또를 판매했기 때문에 
`현 시점의 내 구매 갯수와 총 가격`을 알고 있고, `당첨과 관련된 정보에 대해서도 물어볼 수 있으니` 
Vendor 가 수익률을 계산하도록 했어요. 

현실적인 흐름(?) 을 조금 더 고려했던 것 같습니다.

Reply:
> Vendor -> 로또를 직접 구매(편의점 등)
Lottery - > 로또 추첨, 공시(로또 기금 재단)

역할에 대한 깊은 고민을 해보셨군요! 👍 
자연스러운 흐름에서`Vendor`가 적합한 역할이라면 좋습니다 :)

저는 좀 더 책임을 잘게 나눈다 라는 측면에서 접근을 해보았는데요,
```suggestion
    public BigDecimal yield(Ranks ranks) {
        return ranks.calculateYield();
     }
```
그이유는  `totalMoney, size`를 `Ranks` 가 가지고 있기 때문이에요,
그렇다면, `Vendor` 가 수익율를 반환해주는건 맞지만 계산은 `Ranks`에게 맏길수 있지않을까 생각해봤어요~

Reply:
@wooobo  
의견 감사합니다. 좋은 접근이라고 생각합니다. 👍  다음 단계에서 같이 반영해보겠습니다. :)